### PR TITLE
IW-2015 | Add support for tracking pixels in sponsored content

### PR DESCRIPTION
**Pull request by bkoval**
@Wikia/iwing

### CODE REVIEW

**Comment by bkoval** [comment]
Test fail seems unrelated.

File: extensions/wikia/Recirculation/js/recirculation.js [hunk 1]
```diff
@@ -90,6 +90,20 @@ require([
 		});
 	}
 
+	function insertTrackingPixel(pixelContent, pixelType) {
+		var documentFragment = document.createDocumentFragment();
```
Comment (latata) [inline extensions/wikia/Recirculation/js/recirculation.js]: `documentFragment` is not needed here.

**Review by latata — APPROVED** [review]

File: extensions/wikia/CommunityPage/models/CommunityPageShortPagesCardModel.class.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-class CommunityPageShortPagesCardModel {
+class xCommunityPageShortPagesCardModel {
```
Comment (mszabo-wikia) [inline extensions/wikia/CommunityPage/models/CommunityPageShortPagesCardModel.class.php]: `x`?

**Review by mszabo-wikia — DISMISSED** [review]

File: extensions/wikia/CommunityPage/models/CommunityPageShortPagesCardModel.class.php (hunk 2 above)
Comment (bkoval) [reply extensions/wikia/CommunityPage/models/CommunityPageShortPagesCardModel.class.php]: Lol, pre-commit typo, good catch!

**Review by mszabo-wikia — APPROVED** [review]

**Pull request merged**